Utilities/MCStepLogger/include/MCStepLogger/AnalysisHelper.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * some helper functions which can be used by the analyses and by the AnalysisManager
+ */
+#ifndef ANALYSIS_HELPER_H_
+#define ANALYSIS_HELPER_H_
+
+#include <unordered_map>
+#include <vector>
+#include <string>
+
+class TH1;
+
+namespace o2
+{
+namespace mcstepanalysis
+{
+
+class AnalysisHelper
```

Comment:
you probably don't need a class here. consider using `analysishelper` namespace

Reply:
done